runner/run_tg.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, settings):
         super().__init__(settings['tg'])
         try:
             self.token = self.settings['token']
-            self.updater = Updater(token=self.token)
+            self.updater = Updater(token=self.token, use_context=True)
```

Comment:
![image](https://user-images.githubusercontent.com/7874195/64957659-1f19f680-d896-11e9-8d8b-6f6f2930ac42.png)
No need to do this